understanding/20/sign-language-prerecorded.html

```diff
@@ -51,25 +51,18 @@ <h2>Examples of Sign Language (Prerecorded)</h2>
          
          <li> 
             
-            <strong> Example 1.</strong> A corporation is making an important announcement to all of its employees. The meeting
-            will be held in the main headquarters and streamed to the Web. A sign language interpreter
-            is provided at the meeting location. The live video includes a full view of the sign
-            language interpreter as well as the person presenting.
+            <strong> Example 1.</strong> A corporation is making an important announcement to all of its employees. The announcement
+            will be held in the main headquarters and later streamed to the Web. A sign language interpreter
+            is provided at the announcement location for the employees that are present in the meeting room. For the Web version of
+            the announcement, the sign language interpreter is shown/superimposed in the corner of the display.
          </li>
          
          <li> 
             
-            <strong> Example 2.</strong> The same announcement described in example 1 is also Webcast to remote employees.
-            Since there is only one display available for this, the sign language interpreter
-            is shown in the corner of the display.
-         </li>
-         
-         <li> 
-            
-            <strong> Example 3.</strong> A university is providing an on-line version of a particular lecture by creating
+            <strong> Example 2.</strong> A university is providing an on-line version of a particular lecture by creating
```

Comment:
While you're at it, maybe this:

```suggestion
            <strong> Example 2.</strong> A university is providing an online version of a particular lecture by creating
```

Reply:
good call